error.go

```diff
@@ -2,38 +2,42 @@
 // and efficient error handling.
 package microerror
 
-import (
-	"fmt"
-
-	"github.com/juju/errgo"
-)
-
 var (
-	// MaskAny is a simple error masker. Masked errors act as tracers within the
-	// source code. Inspecting an masked error shows where the error was passed
-	// through within the code base. This is gold for debugging and bug huntin.
-	MaskAny = errgo.MaskFunc(errgo.Any)
+	Default = NewErrgoHandler(ErrgoHandlerConfig{
+		CallDepth: 1,
```

Comment:
I'd like to give ability to change global implementation the library user.
Say someone wants to use `operatorkit` which is using `microerror`, then code could look like this.

```
package usercode

import (
	"github.com/giantswarm/microerror"
	"github.com/giantswarm/operatorkit/tpr"
)

func a() {
	microerror.Default = newUsersCustomErrorHandler()

	tpr.New()
}
```